### ISSUE TITLE
DISCUSSION: Hosting

### ISSUE THREAD

**Issue by msolefonte** [ChaosReminder/WE-REMEMBER#1]
Copy paste of the current conversation at Discord:

**Wolfy**

> Hi everyone
> I missed previous discussion, but I think a self-hosted GitHub page may be enough
> Let just people push the names via Pull Requests
> No need to host a db/website
> If we have to rely on paying every month for hosting, it will eventually go down
> Check this out: https://pages.github.com/

**triss_and_yen**

> Just read through both the channels. I am really loving the ideas!
> - Looks like we have a repo https://github.com/ChaosReminder/WE-REMEMBER
> - There are some ideas about what we want to do: headstones, poneglyph, flying lanterns
> - There were concerns about spam on both the channels, and the consensus was that we can have a simple filter using CAPTCHA and limiting submissions for now, while we focus on the core functionality. Along with people volunteering their time to manually approve submissions. 
> 
> Just putting this here for more visibility. Did we end up agreeing on what idea are we going with?

**Wolfy**

> I also think a GitHub organization is more suited for this than a personal account
> Something like one-piece-memorial/we-remember
> If we rely on Pull Requests we will not have to worry about captchas. Hopefuly changes will not be that frequent

**triss_and_yen**

> I think people wanted to make it open to submissions for everyone

**Wolfy**

> Pull Requests are open, they just have to be accepted by a member of the organization
> Hence filtering spam

**triss_and_yen**

> No, I mean it might not be something any person could/would submit
> Unless we come up with a way of automating a form submission to a PR 
> edit: remove static part

**Wolfy**

> I do not expect (and hope) that we will get that many submissions. Even an email would be a good idea
> If we have to create and host a database, plus all the backend for submissions, it will end up being too big
> Plus the cost of maintaing it
> Even if it is something like 5€/month, we have to think on the long term imo

**triss_and_yen**

> Fair enough, it might also be a lot easier for any of us to maintain then I guess

**Wolfy**

> Yup
> Long term is more important than overengineering here
> Steps, imo:
> 
> 1. Create an email account. Something like onepiecememorial@gmail.com
> 2. Use that account to create a GitHub organization:  https://github.com/organizations/plan
>     - Free Plan
>     - Name it something like onepiecememorial
> 3. Create a IO repo, something like "weremember.github.io"

**ItsJae**

> We should have a discussion of a gameplan when more devs are online and get the plans solidified. Think everyone is sleeping rn

**veebee**

> Since most of us are in different time zones (at least I think we would be) is there a way we can make the plan a little async? Perhaps through an issue/discussion in GitHub?

**Comment by msolefonte** [comment]
TL;DR I personally think we should not overengineer this by hosting a database and a backend. I know that there a lot of developers eager to code and help but it would be better to just stick to something more functional and future-proof. 

If we have to pay for a host we will have to rely on donations and on particulars holding money. I think this should not be about any cost/benefit, just about showing respect to those who are not here anymore.

**Comment by vritant24** [comment]
I agree on avoiding overengineering and keeping it static at least to start with.
I do think we can use a form (google form, survey monkey etc) for submissions instead as it provides more structure and the results can be accessed by many personal accounts.
It also gives us the ability to embed the form within the static website with little work.

**Comment by msolefonte** [comment]
Move to https://github.com/onepiecememorial/discussion/discussions/2
